app/views/applications/edit.html.haml

```diff
@@ -47,53 +44,53 @@
       = profile_fields.hidden_field :id
 
       %tr
-        %td= profile_fields.label :twitter, 'Twitter username'
+        %td= profile_fields.label :twitter, 'Twitter username (optional)'
         %td= profile_fields.text_field :twitter
 
       %tr
-        %td= profile_fields.label :facebook, 'Facebook url'
+        %td= profile_fields.label :facebook, 'Facebook URL (optional)'
         %td= profile_fields.text_field :facebook
 
       %tr
-        %td= profile_fields.label :website, 'Website url'
+        %td= profile_fields.label :website, 'Website URL (optional)'
         %td= profile_fields.text_field :website
 
       %tr
-        %td= profile_fields.label :linkedin, 'LinkedIn url'
+        %td= profile_fields.label :linkedin, 'LinkedIn URL (optional)'
         %td= profile_fields.text_field :linkedin
 
       %tr
-        %td= profile_fields.label :blog, 'Blog url'
+        %td= profile_fields.label :blog, 'Blog URL (optional)'
         %td= profile_fields.text_field :blog
 
       %tr
         %td
           = profile_fields.label :summary, 'Tell us a little about yourself!'
-          %small (2000 character maximum)
+          %small In this question, we're just curious to get an overall sense of your story. We don't evaluate applications based on how accomplished or successful a person is. Some sentences or a paragraph would be great; for all of these questions, we love getting to read some detail but don't want to burden you with writing a huge essay. Feel free to include extra URLs if you like. If you don't have a public online presence (such as Twitter or a website), which is fine, it can help us if you add a little extra here. (2000 character maximum.)
```

Comment:
Putting the helper text in a separate paragraph (instead of inline with the question) will probably make the essay questions a little more visually clear